testingutils/setup.go

```diff
@@ -255,9 +233,6 @@ func BuildIntegrationTestingContext() map[string]interface{} {
 	StartPOAGeth()
 	StartCentChain()
 	StartBridge()
-	AddBalanceToBridgeAccount()
```

Comment:
cool, so since the startbridge script already bundles the add_balance, we do not need to add it here. What about the `RunSmartContractMigrations()`, when we only execute a single integration test? It will not be executed unless goes through the test_wrapper, right?

Reply:
nah, it will still go through if the bridge is not running